doc/nrf/known_issues.rst

```diff
@@ -364,6 +364,14 @@ Secure Partition Manager and application building together
 DFU and FOTA
 ============
 
+.. rst-class:: v1-4-0
+
+Socket API calls may hang when using :ref:`lib_download_client`
+  When using the :ref:`lib_download_client` library with HTTP (without TLS), the application might not process incoming fragments fast enough, which can starve BSD library buffers and make calls to BSD library hang.
+  Samples and applications that are affected include those using :ref:`lib_download_client` to download files via HTTP, or :ref:`lib_fota_download` with modem updates enabled.
```

Comment:
```suggestion
Socket API calls may hang when using Download client
  When using the :ref:`lib_download_client` library with HTTP (without TLS), the application might not process incoming fragments fast enough, which can starve the :ref:`nrfxlib:bsdlib` buffers and make calls to BSD library hang.
  Samples and applications that are affected include those that use :ref:`lib_download_client` to download files through HTTP or those that use :ref:`lib_fota_download` with modem updates enabled.
```
Please re-add JIRA ticket for this issue. JIRA tickets are now a must-have for each new known issue (workaround is optional).